### ISSUE TITLE
ontowiki status

### ISSUE THREAD

**Issue by sharpaper** [AKSW/OntoWiki#440]
Hi, sorry I don't know if this is the best to place to ask, anyway I was wondering if ontowiki is still developed and maintained?

**Comment by stegro** [comment]
I wonder, too. For example for me, the demo http://master.demo.ontowiki.eu/ seems to be broken (I see a 0 bytes page) and I have the feeling this should not happen for an active project.

**Comment by white-gecko** [comment]
You are right. Currently there is nobody further developing the existing OntoWiki code base. The code still runs, but only on PHP 5 which does not receive updates anymore.

**Comment by miku** [comment]
Today, I discovered https://www.repostatus.org/ - maybe useful for setting expectations?

**Comment by WolfgangFahl** [comment]
Just found out that http://ontowiki.net/ has a link to stackoverflow but the tag didn't even exist. I just fixed that a minute ago and also did an initial https://stackoverflow.com/tags/ontowiki/info

It would be good if question on stackoverflow would actually be answered.

**Comment by white-gecko** [comment]
I see there is still interest in OntoWiki. To collect the use cases and so on, I have opened this repository. I would like to read your stories: https://github.com/AKSW/OntoWiki.UseCases
